src/plugins/index.js

```diff
@@ -106,6 +107,12 @@ export function getActivityTrackingPlugin({ agent, isProd }) {
 	};
 }
 
+export function getClickTrackingPlugin() {
```

Comment:
It's mostly just to have a consistent pattern for registering the plugin and keeping the main `getPlugins` function succinct - calling a function for each plugin from the `getPlugins` function just makes it easier to add/change 'options' for each plugin over time.